Tests/Unit/LoRaWanTests/EuiTests.cs

```diff
@@ -127,10 +174,22 @@ public void TryParse_Input_Is_Case_Insensitive()
         [Fact]
         public void TryParse_Returns_False_When_Input_Is_Invalid()
         {
-            var succeeded = DevEui.TryParse("foobar", out var result);
+            var succeeded = TryParse("foobar", out var result);
             Assert.False(succeeded);
-            Assert.True(result == default);
+            Assert.Equal(default, result);
         }
+
+        [Fact]
+        public void String_Interpolation_Success_Case()
+        {
+            var result = $"N = {Subject:N}, I = {Subject:I}";
+            Assert.Equal("N = 0123456789ABCDEF, I = 0123:4567:89AB:CDEF", result);
+        }
```

Comment:
I would recommend testing all formats here in the interpolation.

Alternatively, you could turn this into a theory, borrowing almost the same data as you have for `ToString_Returns_Correctly_Formatted_String`:

```suggestion
        [Theory]
        [InlineData("{0}", "01-23-45-67-89-AB-CD-EF")]
        [InlineData("{0:G}", "01-23-45-67-89-AB-CD-EF")]
        [InlineData("{0:D}", "01-23-45-67-89-AB-CD-EF")]
        [InlineData("{0:g}", "01-23-45-67-89-ab-cd-ef")]
        [InlineData("{0:d}", "01-23-45-67-89-ab-cd-ef")]
        [InlineData("{0:E}", "01:23:45:67:89:AB:CD:EF")]
        [InlineData("{0:e}", "01:23:45:67:89:ab:cd:ef")]
        [InlineData("{0:I}", "0123:4567:89AB:CDEF")]
        [InlineData("{0:i}", "0123:4567:89ab:cdef")]
        [InlineData("{0:N}", "0123456789ABCDEF")]
        [InlineData("{0:n}", "0123456789abcdef")]
        public void String_Interpolation_Success_Case(string format, string expected)
        {
            var result = string.Format(CultureInfo.InvariantCulture, format, Subject);
            Assert.Equal(expected, result);
        }
```